openquake/hazardlib/gsim/kanno_2006.py

```diff
@@ -139,17 +139,18 @@ def get_mean_and_stddevs(self, sites, rup, dists, imt, stddev_types):
         log_mean += self._compute_site_amplification(sites, coeffs)
 
         # retrieve standard deviations
-        log_stddevs = self._get_stddevs(coeffs, stddev_types)
+        log_stddevs = self._get_stddevs(coeffs, sites.vs30.size, stddev_types)
 
         # convert from common to natural logarithm
-        ln_mean = log_mean*np.log(10)
-        ln_stddevs = log_stddevs*np.log(10)
+        LOG10 = np.log(10)
```

Comment:
No no Robin, what's the point of inlining the constant? Only if you define it at top level you can avoid repeating the calculation!